src/databaseClustering.py

```diff
@@ -104,8 +104,8 @@ def cluster_sentences(language_name: str, save_path: str, sentences: List[str],
 
 
 a = database()
-file_path= r'./corpus/word2vecmodel/'
-language_name = 'English'
+file_path = './corpus/word2vecmodel/'
```

Comment:
We have to store the model in the repo, so that we can call it without having to change the file path for each user.